.github/guidelines/LABELING_GUIDELINES.md

```diff
@@ -1,17 +1,20 @@
 # PR Labeling Guidelines
 To maintain a consistent and efficient development workflow, we have set specific label guidelines for all pull requests (PRs). Please ensure you adhere to the following instructions:
 
-### Mandatory Labels:
-- **Internal Developers**: Every PR raised by an internal developer must include a label prefixed with `team-` (e.g., `team-extension-ux`, `team-extension-platform`, etc.). This indicates the respective internal team responsible for the PR.
+### Mandatory team labels:
+- **Internal Developers**: Every PR raised by an internal developer must include a label prefixed with `team-` (e.g., `team-mobile-ux`, `team-mobile-platform`, etc.). This indicates the respective internal team responsible for the PR.
```

Comment:
```suggestion
- **Internal Developers**: Every PR raised by an internal developer must include a label prefixed with `team-` (e.g., `team-extension-ux`, `team-extension-platform`, etc.). This indicates the respective internal team responsible for the PR.
```

Reply:
Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/20651/commits/021be0ca2ee2f31f70a29b0e1a1d493d229a9099).